languages/translations/fr.json

```diff
@@ -118,367 +118,367 @@
                 "default": "S’aligner légèrement à gauche",
-                "name": "S’aligner légèrement à gauche pour rester sur {way_name}",
-                "destination": "S’aligner légèrement à gauche en direction de {destination}"
+                "name": "S’aligner légèrement à gauche pour rester sur {way_name:article}",
+                "destination": "S’aligner légèrement à gauche en direction {destination:elision}"
             },
             "slight right": {
                 "default": "S’aligner légèrement à droite",
-                "name": "S’aligner légèrement à droite pour rester sur {way_name}",
-                "destination": "S’aligner légèrement à droite en direction de {destination}"
+                "name": "S’aligner légèrement à droite pour rester sur {way_name:article}",
+                "destination": "S’aligner légèrement à droite en direction {destination:elision}"
             },
             "uturn": {
                 "default": "Faire demi-tour",
-                "name": "Faire demi-tour et continuer sur {way_name}",
-                "destination": "Faire demi-tour en direction de {destination}"
+                "name": "Faire demi-tour et continuer sur {way_name:article}",
+                "destination": "Faire demi-tour en direction {destination:elision}"
             }
         },
         "depart": {
             "default": {
                 "default": "Rouler vers {direction}",
-                "name": "Rouler vers {direction} sur {way_name}",
-                "namedistance": "Rouler vers {direction} sur {way_name} sur {distance}"
+                "name": "Rouler vers {direction} sur {way_name:article}",
+                "namedistance": "Rouler vers {direction} sur {way_name:article} sur {distance}"
             }
         },
         "end of road": {
             "default": {
                 "default": "Tourner {modifier}",
-                "name": "Tourner {modifier} sur {way_name}",
-                "destination": "Tourner {modifier} en direction de {destination}"
+                "name": "Tourner {modifier} sur {way_name:article}",
+                "destination": "Tourner {modifier} en direction {destination:elision}"
             },
             "straight": {
                 "default": "Continuer tout droit",
-                "name": "Continuer tout droit sur {way_name}",
-                "destination": "Continuer tout droit en direction de {destination}"
+                "name": "Continuer tout droit sur {way_name:article}",
+                "destination": "Continuer tout droit en direction {destination:elision}"
             },
             "uturn": {
                 "default": "Faire demi-tour à la fin de la route",
-                "name": "Faire demi-tour à la fin de la route {way_name}",
-                "destination": "Faire demi-tour à la fin de la route en direction de {destination}"
+                "name": "Faire demi-tour à la fin de la route {way_name:article}",
```

Comment:
The correct translation would be `Faire demi-tour à la fin {way_name:elision}`.

Reply:
Correction: `Faire demi-tour à la fin de {way_name:article}`.

Reply:
So `Faire demi-tour à la fin de le boulevard Saint-Germain` not `Faire demi-tour à la fin du boulevard Saint-Germain`?